cmake/onnxruntime.cmake

```
@@ -98,7 +98,7 @@ else()
 endif()
 
 add_dependencies(onnxruntime onnxruntime_generate_def ${onnxruntime_EXTERNAL_DEPENDENCIES})
-target_include_directories(onnxruntime PRIVATE ${ONNXRUNTIME_ROOT})
+target_include_directories(onnxruntime PRIVATE ${ONNXRUNTIME_ROOT} PUBLIC "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/onnxruntime;${CMAKE_INSTALL_INCLUDEDIR}/onnxruntime/core/session>")
```

```suggestion
target_include_directories(onnxruntime PRIVATE ${ONNXRUNTIME_ROOT} PUBLIC "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/onnxruntime;${CMAKE_INSTALL_INCLUDEDIR}/onnxruntime/core>")
```
Contents of session are installed into `${CMAKE_INSTALL_INCLUDEDIR}/onnxruntime`.